week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,22 @@
+fetch('https://xkcd.now.sh/?comic=latest')
+    .then(function(response) {
+        return response.json();
+    })
+    .then(function(obj) {
+        console.log(obj);
```

Comment:
Super! You didn't forget this :)

Reply:
When you log an error, usually we write to the error log to indicate the type of the issue. The console has a bunch of features among others it can separate so-called log levels such as Log, Info, Debug, Warn and Error. Check them out and see the differences in the console.
https://www.youtube.com/watch?v=RLjhC5r1mRo